File: backend/src/main/java/org/globe42/dao/PersonDaoCustom.java
package org.globe42.dao;

/**
 * Custom methods of {@link PersonDao}
 * @author JB Nizet
 */
public interface PersonDaoCustom {

    /**
     * Gets the next sequence number for the given mediation code letter (which should be 'a' to 'z' or 'A' to 'Z')
     */
    int nextMediationCode(char letter);
}

File: backend/src/main/kotlin/org/globe42/dao/PersonDaoCustom.kt
package org.globe42.dao

/**
 * Custom methods of [PersonDao]
 * @author JB Nizet
 */
interface PersonDaoCustom {

    /**
     * Gets the next sequence number for the given mediation code letter (which should be 'a' to 'z' or 'A' to 'Z')
     */
    fun nextMediationCode(letter: Char): Int
}
